### PR TITLE
Tcpdump 2020 07

### PR DESCRIPTION
**Pull request by mcr**
Two CVEs from 2020 so far.

### CODE REVIEW

**Comment by cve-team** [comment]
mcr is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

**Comment by cristina479** [comment]
There is no such thing as a `RESERVED for TCPDUMP` state. The only valid states are `RESERVED`, `PUBLIC`, or `REJECT`. Please correct this. Also, Please revert the description of the RESERVED CVE ids to the original message: 
> "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."

File: 2020/8xxx/CVE-2020-8038.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
     "CVE_data_meta": {
         "ID": "CVE-2020-8038",
         "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "RESERVED for TCPDUMP"
```
Comment (cristina479) [inline 2020/8xxx/CVE-2020-8038.json:8]: Change to `RESERVED`

File: 2020/8xxx/CVE-2020-8038.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "** RESERVED for TCPDUMP ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
```
Comment (cristina479) [inline 2020/8xxx/CVE-2020-8038.json:14]: Change to
`** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided.`

File: 2020/8xxx/CVE-2020-8036.json [hunk 3]
```diff
@@ -4,15 +4,71 @@
...
+
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "The tok2strbuf was used by the SOME/IP dissector in an unsafe way, but this code was fixed before any release occured."
```
[the remark this answers is not in the document]
Comment (cristina479) [reply 2020/8xxx/CVE-2020-8036.json:49]: If the product was never available why would a CVE ID be assigned to it? 

Per CNA Rules 3.0, Section 7.4: Requirements for Assigning a CVE ID, item 7.4.7 say "CNAs SHOULD NOT assign CVE IDs to vulnerabilities in products that are not publicly available or licensable."

If the affected product and version of CVE-2020-8036 fits the description in 7.4.7, is fair to say that it should not be assigned a CVE ID. Please advice.

**Comment by mcr** [comment]
> There is no such thing as a `RESERVED for TCPDUMP` state. The only valid states are `RESERVED`, `PUBLIC`, or `REJECT`. Please correct this. Also, Please revert the description of the RESERVED CVE ids to the original message:
> 
> > "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."

okay. I wanted to tag the numbers which were assigned to us in some way.
How do you propose that we do this?

File: 2020/8xxx/CVE-2020-8036.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (mcr) [reply 2020/8xxx/CVE-2020-8036.json:49]: All code is publicly available.  It's open source. 
People can fork the project at any time, and without prior arrangement and do their own thing.

File: 2020/8xxx/CVE-2020-8036.json [hunk 4]
```diff
@@ -4,15 +4,71 @@
...
-}
\ No newline at end of file
+    },
+    "problemtype": {
+        "problemtype_data": [
+            {
+                "description": [
+                    {
+                        "lang": "eng",
+                        "value": "n/a"
```
[the remark this answers is not in the document]
Comment (mcr) [reply 2020/8xxx/CVE-2020-8036.json:59]: I have reviewed the schema, and I can not determine the set of values which I can put here.

File: 2020/8xxx/CVE-2020-8036.json [hunk 5]
```diff
@@ -4,15 +4,71 @@
...
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "n/a",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "n/a"
```
[the remark this answers is not in the document]
Comment (mcr) [reply 2020/8xxx/CVE-2020-8036.json]: added.

**Comment by mcr** [comment]
I have pushed a revised 8036 and 8037.  Please advise if I got anything else wrong, as these will be used as templates for future reports.

File: 2020/8xxx/CVE-2020-8036.json [hunk 6]
```diff
@@ -4,15 +4,72 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:4]: ('Expecting value: line 72 column 9 (char 2153)',)

File: 2020/8xxx/CVE-2020-8037.json [hunk 7]
```diff
@@ -4,15 +4,71 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: ('Expecting value: line 72 column 9 (char 2094)',)

File: 2020/8xxx/CVE-2020-8036.json [hunk 8]
```diff
@@ -4,15 +4,69 @@
...
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:4]: ('Expecting value: line 69 column 9 (char 2074)',)

File: 2020/8xxx/CVE-2020-8037.json [hunk 9]
```diff
@@ -4,15 +4,68 @@
...
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: ('Expecting value: line 69 column 9 (char 2015)',)

File: 2020/8xxx/CVE-2020-8036.json [hunk 10]
```diff
@@ -1,18 +1 @@
-{
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: 'ASSIGNER' is a required property

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: 'vendor_name' is a required property

File: 2020/8xxx/CVE-2020-8037.json (hunk 9 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: ('Expecting value: line 69 column 9 (char 2015)',)

File: 2020/8xxx/CVE-2020-8036.json (hunk 10 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: ['security@tcpdump.org', '2020-02-29T00:00:00.000Z'] is not of type 'string'

File: 2020/8xxx/CVE-2020-8037.json (hunk 9 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:4]: ('Expecting value: line 69 column 9 (char 2015)',)

File: 2020/8xxx/CVE-2020-8036.json (hunk 10 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: Invalid source.

File: 2020/8xxx/CVE-2020-8037.json [hunk 11]
```diff
@@ -1,18 +1 @@
-{
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:1]: Invalid source.

File: 2020/8xxx/CVE-2020-8036.json (hunk 10 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: Invalid source.

File: 2020/8xxx/CVE-2020-8037.json (hunk 11 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8037.json:1]: 'references' is a required property

File: 2020/8xxx/CVE-2020-8036.json (hunk 10 above)
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8036.json:1]: Invalid source.

**Pull request closed without merging**